js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -156,38 +161,53 @@ const KeyboardNavigationController = core.ViewController.inherit({
     },
 
     _initDocumentHandlers: function() {
-        const that = this;
         const document = domAdapter.getDocument();
 
-        that._documentClickHandler = that.createAction(function(e) {
+        this._documentClickHandler = this.createAction((e) => {
             const $target = $(e.event.target);
-            const isCurrentRowsViewClick = that._isEventInCurrentGrid(e.event) && $target.closest(`.${that.addWidgetPrefix(ROWS_VIEW_CLASS)}`).length;
+            const isCurrentRowsViewClick = this._isEventInCurrentGrid(e.event) && $target.closest(`.${this.addWidgetPrefix(ROWS_VIEW_CLASS)}`).length;
             const isEditorOverlay = $target.closest(`.${DROPDOWN_EDITOR_OVERLAY_CLASS}`).length;
-            if(!isCurrentRowsViewClick && !isEditorOverlay) {
-                that._resetFocusedCell();
+            const columnsResizerController = this.getController('columnsResizer');
+            const isColumnResizing = !!columnsResizerController && columnsResizerController.isResizing();
+            if(!isCurrentRowsViewClick && !isEditorOverlay && !isColumnResizing) {
```

Comment:
Why is `!isColumnResizing` condition added?

Reply:
Because we should not reset focused cell position when a user resizes columns.